src/rgw/rgw_tag.cc

```diff
@@ -10,12 +10,12 @@
 #include "rgw_tag.h"
 #include "rgw_common.h"
 
-bool RGWObjTags::add_tag(const string&key, const string& val){
-  return tag_map.emplace(std::make_pair(key,val)).second;
+void RGWObjTags::add_tag(const string&key, const string& val){
```

Comment:
is there a space between "string&" and "key" above?  I didn't think that was possible to omit

Reply:
oh! the original code also didnt have the space, but I will fix it.